src/common/services/session/sessionEnforcer.service.js

```diff
@@ -98,8 +98,7 @@ const SessionEnforcerService = /* @ngInject */ function ($window, orderService,
       })
 
       if (angular.isUndefined(modal)) {
-        const modalType = find(enforced, { mode: EnforcerModes.donor }) ? 'register-account' : 'sign-in'
-        modal = sessionModalService.open(modalType, {
+        modal = sessionModalService.open('register-account', {
           backdrop: 'static',
           keyboard: false
         }).result
```

Comment:
Should `const modalType = find(enforced, { mode: EnforcerModes.donor })` this define if the register-account modal is minimal

Reply:
I'm not sure what you're asking. But now the `register-account` modal now defaults to minimal mode (`welcomeBack` is falsy), if that's what you mean.